bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -145,22 +146,8 @@ impl Room {
         self.inner.state().into()
     }
 
-    /// Is there a non expired membership with application "m.call" and scope
-    /// "m.room" in this room.
-    pub fn has_active_room_call(&self) -> bool {
-        self.inner.has_active_room_call()
-    }
-
-    /// Returns a Vec of userId's that participate in the room call.
-    ///
-    /// matrix_rtc memberships with application "m.call" and scope "m.room" are
-    /// considered. A user can occur twice if they join with two devices.
-    /// convert to a set depending if the different users are required or the
-    /// amount of sessions.
-    ///
-    /// The vector is ordered by oldest membership user to newest.
-    pub fn active_room_call_participants(&self) -> Vec<String> {
-        self.inner.active_room_call_participants().iter().map(|u| u.to_string()).collect()
+    pub fn active_room_call(&self) -> Option<Arc<RoomCall>> {
+        self.inner.active_room_call().map(|c| Arc::new(RoomCall { inner: c }))
```

Comment:
I think it'll have to become something like this:

```rust
fn active_sessions(&self, filter: SessionFilter) -> HashMap<SessionId, ActiveSession>
```

Where the `SessionFilter` will be an enum of either using an app filter or a session ID filter.

That being said, it looks like it would be quite cumbersome to use this API just to fetch the active room call.

I think perhaps `active_room_call()` is not as bad (it could be viewed as a helper function that fetches the right RTC sessions in addition to a more generic `active_sessions()` should we ever need this function).